AdminWebsite/AdminWebsite.UnitTests/Controllers/StaffMemberControllerTest.cs

```diff
@@ -0,0 +1,93 @@
+﻿using AdminWebsite.Controllers;
+using AdminWebsite.UnitTests.Helper;
+using BookingsApi.Client;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Contract.Responses;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+
+namespace AdminWebsite.UnitTests.Controllers
+{
+
+    public class StaffMemberControllerTest
+    {
+        private StaffMemberController _controller;
+        private Mock<IBookingsApiClient> _bookingsApiClient;
+        private List<PersonResponse> _response;
+
+        [SetUp]
+        public void Setup()
+        {
+            _bookingsApiClient = new Mock<IBookingsApiClient>();
+
+            _controller = new StaffMemberController(_bookingsApiClient.Object, JavaScriptEncoder.Default);
+
+            _response = new List<PersonResponse>
+            {
+                new PersonResponse
+                {
+                  Id = Guid.NewGuid(),
+                  ContactEmail = "adoman@hmcts.net",
+                  FirstName = "Adam",
+                  LastName = "Mann",
+                  TelephoneNumber ="111222333",
+                  Title = "Ms",
+                  MiddleNames = "No",
+                  Username = "adoman@hmcts.net"
+                }
+            };
+
+        }
+
+        [Test]
+        public async Task GetStaffMembersBySearchTerm_Returns_StaffMembers_WhenMatch_SearchTerm()
+        {
+            _response.Add(new PersonResponse
+            {
+                Id = Guid.NewGuid(),
+                ContactEmail = "",
+                FirstName = "Jack",
+                LastName = "Mann",
+                TelephoneNumber = "",
+                Title = "Mr",
+                MiddleNames = "No",
+                Username = "jackman@judiciary.net"
+            });
+            _bookingsApiClient.Setup(x => x.GetStaffMemberBySearchTermAsync(It.IsAny<SearchTermRequest>()))
+                              .ReturnsAsync(_response);
+
+            var searchTerm = "ado";
+            var result = await _controller.GetStaffMembersBySearchTerm(searchTerm);
+
+            var okRequestResult = (OkObjectResult)result.Result;
+            okRequestResult.StatusCode.Should().NotBeNull();
```

Comment:
I can see you have used a different type of assertions for bad request, does that not work with ok result?

Reply:
it will work, but if you read the next line I am consuming the content of the response, which is not the case when it comes to bad requests.